webapp_utils/errata.py

```diff
@@ -167,9 +167,12 @@ def _errata_exists(self, errata):
         """Checks if given errata is in db."""
         db_connection = self.db_pool.get_connection()
         with db_connection.get_cursor() as cursor:
-            cursor.execute("""select name from errata
-                              where name = '%s'""" % errata)
-            errata_detail = cursor.fetchone()
+            try:
+                cursor.execute("""select name from errata
+                                  where name = '%s'""" % errata)
+                errata_detail = cursor.fetchone()
+            except Exception as _: # pylint: disable=broad-except
```

Comment:
I think you could use just `except Exception` without naming it if I am not mistaken.
Also, could more specific exception, more related to the issue be used so we don't catch there everything (this can be problematic in the future, as here we would silently consume all the exceptions without logging it).